bittensor/utils/__init__.py

```diff
@@ -263,70 +159,7 @@ def format_error_message(error_message: dict) -> str:
     return f"Subtensor returned `{err_name} ({err_type})` error. This means: `{err_description}`"
 
 
-def create_identity_dict(
-    display: str = "",
-    legal: str = "",
-    web: str = "",
-    riot: str = "",
-    email: str = "",
-    pgp_fingerprint: Optional[str] = None,
-    image: str = "",
-    info: str = "",
-    twitter: str = "",
-) -> dict:
-    """
-    Creates a dictionary with structure for identity extrinsic. Must fit within 64 bits.
-
-    Args:
-        display (str): String to be converted and stored under 'display'.
-        legal (str): String to be converted and stored under 'legal'.
-        web (str): String to be converted and stored under 'web'.
-        riot (str): String to be converted and stored under 'riot'.
-        email (str): String to be converted and stored under 'email'.
-        pgp_fingerprint (str): String to be converted and stored under 'pgp_fingerprint'.
-        image (str): String to be converted and stored under 'image'.
-        info (str): String to be converted and stored under 'info'.
-        twitter (str): String to be converted and stored under 'twitter'.
-
-    Returns:
-        dict: A dictionary with the specified structure and byte string conversions.
-
-    Raises:
-        ValueError: If pgp_fingerprint is not exactly 20 bytes long when encoded.
-    """
-    if pgp_fingerprint and len(pgp_fingerprint.encode()) != 20:
-        raise ValueError("pgp_fingerprint must be exactly 20 bytes long when encoded")
-
-    return {
-        "info": {
-            "additional": [[]],
-            "display": {f"Raw{len(display.encode())}": display.encode()},
-            "legal": {f"Raw{len(legal.encode())}": legal.encode()},
-            "web": {f"Raw{len(web.encode())}": web.encode()},
-            "riot": {f"Raw{len(riot.encode())}": riot.encode()},
-            "email": {f"Raw{len(email.encode())}": email.encode()},
-            "pgp_fingerprint": pgp_fingerprint.encode() if pgp_fingerprint else None,
-            "image": {f"Raw{len(image.encode())}": image.encode()},
-            "info": {f"Raw{len(info.encode())}": info.encode()},
-            "twitter": {f"Raw{len(twitter.encode())}": twitter.encode()},
-        }
-    }
-
-
-def decode_hex_identity_dict(info_dictionary):
-    for key, value in info_dictionary.items():
-        if isinstance(value, dict):
-            item = list(value.values())[0]
-            if isinstance(item, str) and item.startswith("0x"):
-                try:
-                    info_dictionary[key] = bytes.fromhex(item[2:]).decode()
-                except UnicodeDecodeError:
-                    print(f"Could not decode: {key}: {item}")
-            else:
-                info_dictionary[key] = item
-    return info_dictionary
-
-
+# Subnet 24 uses this function
```

Comment:
Is this needed?

Reply:
> Is this needed?

I'd like to leave this here. I marked this function because it is not used in bittensor, but we should keep it because of subnet 24. Does it make sense?